packages/flutter/lib/src/material/toggle_buttons.dart

```diff
@@ -223,6 +224,14 @@ class ToggleButtons extends StatelessWidget {
   /// are active, selected, or disabled.
   final TextStyle textStyle;
 
+  /// Defines the button's size.
+  ///
+  /// Typically used to constrain the button's minimum size.
+  ///
+  /// If this property is null, then
+  /// BoxConstraints(minWidth: 48.0, minHeight: 48.0) is be used.
```

Comment:
Although the 48 is kMinInteractiveDimension I think the documentation is easier to understand the way you have it.